src/components/Fetch/index.js

```diff
@@ -249,12 +236,12 @@ class Fetch extends Component {
     }
     let findRegexp = this.findInput.value;
 
-    if (findRegexp == '') {
+    if (findRegexp === '') {
       this.clearFind();
       return;
     }
 
-    if (findRegexp == this.state.find && caseSensitive == this.state.caseSensitive) {
+    if (findRegexp == this.state.find && caseSensitive === this.state.caseSensitive) { // eslint-disable-line eqeqeq
```

Comment:
Take out the disable line and just change the first == to ===. We'll see what breaks together.